diff --git a/theories/boolp.v b/theories/boolp.v
index 72a074e..1ec8360 100644
--- a/theories/boolp.v
+++ b/theories/boolp.v
@@ -6,6 +6,10 @@
 
 From mathcomp Require Import ssreflect ssrfun ssrbool eqtype choice.
 
+(* needed to enable rewriting with `<->` *)
+(* (there is probably a more specific `Require` for that) *)
+From mathcomp Require all_ssreflect.
+
 (* -------------------------------------------------------------------- *)
 Set   Implicit Arguments.
 Unset Strict Implicit.
