diff --git a/finmap.v b/finmap.v
index d06e8e5..5f1864f 100644
--- a/finmap.v
+++ b/finmap.v
@@ -6,6 +6,8 @@ From mathcomp Require Import ssreflect ssrbool ssrnat eqtype ssrfun seq.
 Set Warnings "notation-incompatible-format".
 From mathcomp Require Import choice path finset finfun fintype bigop.
 
+Require Coq.Program.Tactics.
+
 (*****************************************************************************)
 (* This file provides representations for finite sets over a choiceType K,   *)
 (* finite maps with keys in a choiceType K and the values in an arbitrary    *)
